DualView/src/main/java/com/microsoft/device/display/samples/contentcontext/MapFragment.kt

```diff
@@ -38,37 +33,54 @@ class MapFragment : Fragment() {
         val layout = inflater.inflate(R.layout.fragment_item_detail, container, false)
         val mapView = layout?.findViewById<MapImageView>(R.id.img_view)
 
-        if (arguments != null) {
-            val item = requireArguments().getParcelable<Restaurant>(Restaurant.KEY)
-            if (item != null && item.mapImageResourceId != 0) {
-                mapView?.setImageResource(item.mapImageResourceId)
-            } else {
-                mapView?.setImageResource(R.drawable.unselected_map)
+        selectedViewModel.selectedPosition.observe(
+            viewLifecycleOwner,
+            {
+                if (it != -1) {
```

Comment:
Same here for this `-1`, better having it in a constant with a descriptive name so we know what's that.